packages/dds/sequence/src/intervalIndex/overlappingIntervalsIndex.ts

```diff
@@ -147,7 +147,9 @@ export class OverlappingIntervalsIndex<TInterval extends ISerializableInterval>
 		if (
 			startPos === undefined ||
 			endPos === undefined ||
-			endPos < startPos ||
+			(typeof startPos === "number" && typeof endPos === "number" && endPos < startPos) ||
+			(startPos === "end" && endPos !== "end") ||
+			(startPos !== "start" && endPos === "start") ||
```

Comment:
Unrelated to this bug but I think this bug (better supporting start and end as arguments here) be something we want to backport regardless. Before it worked for _either_ "start" or "end", but not both

Reply:
yeah seems fine to include.